src/Service/AdminOrderService.php

```diff
@@ -54,12 +54,12 @@ public function createConsignments(
         Order                          $order,
         AbstractDeliveryOptionsAdapter $deliveryOptions = null
     ): ConsignmentCollection {
-        $factory    = new ConsignmentFactory($postValues);
-        $collection = $factory->fromOrder($order, $deliveryOptions);
-        $collection->setLinkOfLabels();
+        $factory        = new ConsignmentFactory($postValues);
+        $collection     = $factory->fromOrder($order, $deliveryOptions);
+        $typeTextForLog = $this->exportLabelOrCreateConcept($collection);
```

Comment:
Dit liever inline, of anders de log statement in die functie zelf. Op deze manier is het heel gek, door dit in een tekst variabele te stoppen impliceer je dat er niks gebeurt in de functie. Functies die meerdere dingen doen zijn onduidelijk, die functie doet nu al "shipment of concept aanmaken" maar ook nog eens "melden of er een shipment of een concept aangemaakt is/wordt".